### DISCUSSION TITLE
Using vuestic ui with vue 3 vite typescript

### DISCUSSION THREAD

**Discussion by otatar** [epicmaxco/vuestic-ui#1496] [Q&A] [answered]
Hi there,

I wanted to try this vue 3 vui library and when I've imported it in main.ts (vue 3 typescript vite) I got this:

8:29:42 PM [vite] new dependencies found: vuestic-ui, updating...
8:29:42 PM [vite] Failed to load source map for /node_modules/.vite/chunk-O3VRSDP2.js?v=08dcd487.
✘ [ERROR] No matching export in "node_modules/tslib/tslib.es6.js" for import "__spreadArray"

    node_modules/asva-executors/dist/src/index.esm.js:1:39:
      1 │ ...nds,__awaiter,__generator,__spreadArray}from"tslib";var Executor...
        ╵                              ~~~~~~~~~~~~~

8:29:48 PM [vite] error while updating dependencies:
Error: Build failed with 1 error:
node_modules/asva-executors/dist/src/index.esm.js:1:39: ERROR: No matching export in "node_modules/tslib/tslib.es6.js" for import "__spreadArray"
    at failureErrorWithLog (/home/otatar/Code/vue-vuestic/node_modules/esbuild/lib/main.js:1601:15)
    at /home/otatar/Code/vue-vuestic/node_modules/esbuild/lib/main.js:1247:28
    at runOnEndCallbacks (/home/otatar/Code/vue-vuestic/node_modules/esbuild/lib/main.js:1032:63)
    at buildResponseToResult (/home/otatar/Code/vue-vuestic/node_modules/esbuild/lib/main.js:1245:7)
    at /home/otatar/Code/vue-vuestic/node_modules/esbuild/lib/main.js:1354:14
    at /home/otatar/Code/vue-vuestic/node_modules/esbuild/lib/main.js:664:9
    at handleIncomingPacket (/home/otatar/Code/vue-vuestic/node_modules/esbuild/lib/main.js:761:9)
    at Socket.readFromStdout (/home/otatar/Code/vue-vuestic/node_modules/esbuild/lib/main.js:631:7)
    at Socket.emit (events.js:400:28)
    at addChunk (internal/streams/readable.js:293:12)

Where am i wrong?

**Comment by m0ksem** [comment]
Vuestic works well with vite for me. Can you give more information about your setup? E.g., vuestic-ui version, vite config. 

The only one issue you should face is lack of alias for normalize.css, that can be fixed if you add 
```ts
  resolve: {
    alias: [
        { find: /^~(.*)$/, replacement: '$1' },
    ],
  },
```
to your vite.config.ts.

**Comment by otatar** [comment, marked as answer]
It is interesting that if I use this command to init vue 3 project:
`npm init vue@latest`
and install vuestic-ui I get previous error.

But if scafold vue 3 project using:
`npm create vite@latest`
and following along, and then install vuestic-ui there is no error...
